mlos_bench/mlos_bench/util.py

```diff
@@ -274,33 +274,147 @@ def check_required_params(config: Mapping[str, Any], required_params: Iterable[s
         )
 
 
-def get_git_info(path: str = __file__) -> tuple[str, str, str]:
+def get_git_root(path: str = __file__) -> str:
```

Comment:
The git changes should be merged in #991 first to make it easier to review this PR.